src/worldstate/src/routines/State.cpp

```diff
@@ -7,13 +7,14 @@
 
 #include "routines/State.h"
 
-State::State() {}
+State::State(std::unordered_map<std::string, double>* constants) {
```

Comment:
Similar comment to above, passing in constants like this isn't a great idea.

Reply:
These constants are from the .yaml file we import, was discussed in previous PR. I'm not sure what the proper way to do this is but I think this was the way we agreed on

Reply:
Since we're collecting rosparam it's hard to be sure they actually load properly in the first place. The function call nodehandle.getParams returns a boolean but this can only be checked on runtime